.github/workflows/cd.yml

```
@@ -0,0 +1,36 @@
+    steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: harbor.remystorage.ru
+          username: ${{ secrets.HARBOR_USER }}
+          password: ${{ secrets.HARBOR_TOKEN }}
+      
+      - name: Read version from file
+        id: version
+        run: |
+          VERSION=$(cat version | tr -d ' \n') 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)